src/imars3dv2/filters/crop.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+import numpy as np
+from scipy.ndimage import median_filter
+
+
+def crop(
+    arrays: np.ndarray,
+    crop_limit: tuple = (-1, -1, -1, -1),
```

Comment:
Which version allows you to switch from `Tuple` to `tuple` on the type annotations?

Reply:
the local dev environment is using python 3.10.4

Reply:
I think the feature came in 3.9